Doc/c-api/frame.rst

```diff
@@ -130,3 +130,38 @@ See also :ref:`Reflection <reflection>`.
 .. c:function:: int PyFrame_GetLineNumber(PyFrameObject *frame)
 
    Return the line number that *frame* is currently executing.
+
+
+
+Internal Frames
+---------------
+
+Unless using PEP 523, you will not need this.
+
+.. c:type:: struct _PyInterpreterFrame
```

Comment:
```suggestion
.. c:type:: _PyInterpreterFrame
```

Reply:
The header doesn't include a `typedef`, so I think `struct _PyInterpreterFrame` is correct. Unless we always skip `struct` in the docs for some reason.

Reply:
`_PyInterpreterFrame` is defined in `pycore_frame.h` with `typedef struct _PyInterpreterFrame { ... } _PyInterpreterFrame`.

It looks like we are inconsistent in code about referring to it as `_PyInterpreterFrame*` vs `struct _PyInterpreterFrame*`, but there are plenty of examples of both. Maybe it originally was defined without the `typedef`, but that was changed at some point?

In any case, there aren't any other examples in the docs of using `c:type::` with `struct ...`, and Sphinx seems to not like it *shrug*.

Reply:
It looks like if you prefer to document this as a struct rather than a type, the way to do that would be to use `c:struct::` instead of `c:type::` (and don't explicitly include the `struct` keyword, the `c:struct::` will add it for you in the rendered docs.) See for example `PyCompilerFlags` in `Doc/c-api/veryhigh.rst`.

Reply:
Carl is right, `.. c:struct::` should be preferred. See the docs for reference: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#the-c-domain

A